src/pages/workspace/reportFields/WorkspaceReportFieldsPage.tsx

```diff
@@ -93,6 +98,20 @@ function WorkspaceReportFieldsPage({
 
     const canSelectMultiple = !hasReportAccountingConnections && (isSmallScreenWidth ? selectionMode?.isEnabled : true);
 
+    const selectionFieldList = useMemo(() => {
+        return Object.values(filteredPolicyFieldList).reduce<Record<string, OnyxValueWithOfflineFeedback<PolicyReportField, 'defaultValue' | 'deletable'>>>((acc, reportField) => {
+            const reportFieldKey = getReportFieldKey(reportField.fieldID);
+            acc[reportFieldKey] = reportField;
+            return acc;
+        }, {});
+    }, [filteredPolicyFieldList]);
```

Comment:
I don't see why we would need both `filteredPolicyFieldList` and `selectionFieldList`.  Let's just modify `filteredPolicyFieldList` so it keyed with `getReportFieldKey`